packages/integration-sdk-runtime/src/execution/dependencyGraph.ts

```diff
@@ -306,7 +309,10 @@ export function executeStepDependencyGraph<
      * Errors from an execution handler are caught and used to
      * determine a status code for the step's result.
      */
-    async function executeStep(step: Step<TStepExecutionContext>) {
+    async function executeStep(
+      step: Step<TStepExecutionContext>,
+      wrapper?: StepWrapperFunction<TStepExecutionContext>,
```

Comment:
Technically, you shouldn't need to add `wrapper` to the interface of `executeStep`, since you've already got access to `stepWrapper` in your current scope.

```diff
    async function executeStep(
      step: Step<TStepExecutionContext>,
-     wrapper?: StepWrapperFunction<TStepExecutionContext>,
    ) {
      const { id: stepId } = step;
      
      // ...

        if (status !== StepResultStatus.CACHED) {
-         if (wrapper) {
+         if (stepWrapper) {
-           await wrapper(
+           await stepWrapper(
              step,
              async () => await step.executionHandler(context),
            );
          } else {
            await step.executionHandler(context);
          }